test/fixtures/Integration-Test-A_2147949032/B-Testing_298655373/html-pipe-sees-only-one-variant_2585928325/recording.har

```diff
@@ -133,6 +133,149 @@
           "ssl": -1,
           "wait": 221
         }
+      },
```

Comment:
Files after this are all just updating `recording.har` files to cover helix-markup.yaml, **except** the last file which contains the actual tests for the feature.